partiql-ast/src/main/java/org/partiql/ast/v1/DataType.java

```diff
@@ -400,53 +400,109 @@ public int code() {
         return code;
     }
 
+    @NotNull
+    @Override
+    public String name() {
+        switch (code) {
+            case NULL: return "NULL";
+            case MISSING: return "MISSING";
+            case CHARACTER: return "CHARACTER";
+            case CHAR: return "CHAR";
+            case CHARACTER_VARYING: return "CHARACTER_VARYING";
+            case CHAR_VARYING: return "CHAR_VARYING";
+            case VARCHAR: return "VARCHAR";
+            case CHARACTER_LARGE_OBJECT: return "CHARACTER_LARGE_OBJECT";
+            case CHAR_LARGE_OBJECT: return "CHAR_LARGE_OBJECT";
+            case CLOB: return "CLOB";
+            case STRING: return "STRING";
+            case SYMBOL: return "SYMBOL";
+            case BLOB: return "BLOB";
+            case BINARY_LARGE_OBJECT: return "BINARY_LARGE_OBJECT";
+            case BIT: return "BIT";
+            case BIT_VARYING: return "BIT_VARYING";
+            case NUMERIC: return "NUMERIC";
+            case DECIMAL: return "DECIMAL";
+            case DEC: return "DEC";
+            case BIGINT: return "BIGINT";
+            case INT8: return "INT8";
+            case INTEGER8: return "INTEGER8";
+            case INT4: return "INT4";
+            case INTEGER4: return "INTEGER4";
+            case INTEGER: return "INTEGER";
+            case INT: return "INT";
+            case INT2: return "INT2";
+            case INTEGER2: return "INTEGER2";
+            case SMALLINT: return "SMALLINT";
+            case TINYINT: return "TINYINT";
+            case FLOAT: return "FLOAT";
+            case REAL: return "REAL";
+            case DOUBLE_PRECISION: return "DOUBLE_PRECISION";
+            case BOOLEAN: return "BOOLEAN";
+            case BOOL: return "BOOL";
+            case DATE: return "DATE";
+            case TIME: return "TIME";
+            case TIME_WITH_TIME_ZONE: return "TIME_WITH_TIME_ZONE";
+            case TIMESTAMP: return "TIMESTAMP";
+            case TIMESTAMP_WITH_TIME_ZONE: return "TIMESTAMP_WITH_TIME_ZONE";
+            case INTERVAL: return "INTERVAL";
+            case STRUCT: return "STRUCT";
+            case TUPLE: return "TUPLE";
+            case LIST: return "LIST";
+            case BAG: return "BAG";
+            case SEXP: return "SEXP";
+            case USER_DEFINED: return "USER_DEFINED";
+            default: return "UNKNOWN";
+        }
+    }
+
     @NotNull
     private static final int[] codes = {
```

Comment:
(self-review): I re-arranged the values in this PR to align with the order of the data type constants defined above https://github.com/partiql/partiql-lang-kotlin/blob/v1-migrate-ast/partiql-ast/src/main/java/org/partiql/ast/v1/DataType.java#L12-L71.